.github/workflows/docker-publish.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
I would recommend to keep only the relevant changes to the problem.

Reply:
Aah, agreed with you. My bad :/ Reverted it ([0548fd3](https://github.com/dimitri/pgcopydb/pull/614/commits/0548fd367b2c44d31e51f3c74c1fa671b5a04d89)) and created a new PR for this change. See #620